Dockerfile

```diff
@@ -56,9 +55,13 @@ RUN apt-get update && apt-get -y install \
   python3-gi \
   python3-openssl \
   python3-venv \
-  softhsm2 \
   valgrind \
   wget
 
+RUN wget http://mirrors.kernel.org/ubuntu/pool/universe/s/softhsm2/softhsm2-common_2.2.0-3_amd64.deb && dpkg -i softhsm2-common_2.2.0-3_amd64.deb
+RUN wget http://mirrors.kernel.org/ubuntu/pool/universe/s/softhsm2/libsofthsm2_2.2.0-3_amd64.deb && dpkg -i libsofthsm2_2.2.0-3_amd64.deb
+RUN wget http://mirrors.kernel.org/ubuntu/pool/universe/s/softhsm2/softhsm2_2.2.0-3_amd64.deb && dpkg -i softhsm2_2.2.0-3_amd64.deb
+RUN wget http://mirrors.kernel.org/ubuntu/pool/main/libp/libp11/libp11-2_0.4.7-1_amd64.deb && dpkg -i libp11-2_0.4.7-1_amd64.deb
+RUN wget http://mirrors.kernel.org/ubuntu/pool/main/libp/libp11/libp11-dev_0.4.7-1_amd64.deb && dpkg -i libp11-dev_0.4.7-1_amd64.deb; test 1
```

Comment:
Just curious: what makes these wgets necessary instead of the regular apt-gets? And why the `test 1` at the end?

Reply:
I will explain on the call